JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,29 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  let hours = 0;
+  let minutes = 0;
+  let formatHMS = 0;
+
+  function padLeadingZeros(num) {
+    if (num < 10) {
+      return `0${num}`
+    } else {
+      return num
+    }
+  };
```

Comment:
we'll talk about this later but to improve legibility please add blank lines between `if`, `functions`, `variables`, etc